ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala

```diff
@@ -129,3 +130,37 @@ object FtpCredentials {
    */
   final case class NonAnonFtpCredentials(username: String, password: String) extends FtpCredentials
 }
+
+object SftpIdentity {
+
+  /** Java API */
+  def createEmptyIdentity: Option[SftpIdentity] = None
+
+  /** Java API */
+  def createSftpIdentity(name: String, privateKey: Array[Byte]): Option[SftpIdentity] =
+    Some(SftpIdentity(name, privateKey))
+
+  /** Java API */
+  def createSftpIdentity(name: String, privateKey: Array[Byte], password: Array[Byte]): Option[SftpIdentity] =
+    Some(SftpIdentity(name, privateKey, Some(password)))
+
+  /** Java API */
+  def createSftpIdentity(name: String,
```

Comment:
I think the format on signatures is not following the previous ones along the project. It's just a nitpick but important in the long term.